website/docs/artist_hosts_nuke_tut.md

```diff
@@ -0,0 +1,195 @@
+---
+id: artist_hosts_nuke_tut
+title: Nuke QuickStart
+sidebar_label: Nuke QuickStart
+---
+
+This QuickStart is just a small introduction to what OpenPype can do for you. It attempts to make an overview for compositing artists, and simplifies processes that are better described in specific parts of the documentation.
+
+## Launch Nuke - Shot and Task Context
+OpenPype has to know what shot and task you are working on. You need to run Nuke in context of the task, using Ftrack Action or OpenPype Launcher to select the task and run Nuke.
+
+![Run Nuke From Ftrack](assets/nuke_tut/nuke_RunNukeFtrackAction_p3.png)
+![Run Nuke From Launcher](assets/nuke_tut/nuke_RunNukeLauncher_p2.png)
+
+:::tip Admin Tip - Nuke version
+You can [configure](admin_settings_project_anatomy.md#Attributes) which DCC version(s) will be available for current project in **Studio Settings → Project → Anatomy → Attributes → Applications**
+:::
+
+## Nuke OpenPype menu shows the current context
```

Comment:
I'd move this section under Nuke Initial Setup, it doesn't warrant an h2 header for itself.

Reply:
done